config/200-config.yaml

```diff
@@ -58,3 +58,6 @@ data:
     # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
     #       for now. Use with caution.
     cluster-cert-secret: ""
+
+    # Specifies the amount of time that Kourier waits for the incoming requests.
+    stream_idle_timeout: 300
```

Comment:
a)  We need to document this and the end-to-end timeout settings in general.
For example I assume that users will need to adapt this setting if they also configure revision time out. 
b) There is also `max_stream_duration` from what I read, do we need to set that to the `DefaultMaxRevisionTimeoutSeconds`.

Reply:
Thanks for your comments @skonto
a) Sure, I will add a detailed description of this option. As for `end-to-end timeout`, I guess you mean the `timeoutSeconds` in knative service, now we could simply set it to "0" to disable the timeout check at ingress level. In the future,  it is possible to overwrite `stream_idile_timeout` with `timeoutSecond` if it is longer than `timeoutSecond`
b) As for other timeout configs, I think we could consider them later and implement them in other PRs if needed.